config/stage_test/stage_test.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: stage-test
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      serviceAccountName: stage-test-service-account
+      containers:
+        - image: quay.io/cloudservices/floorist-operator:latest
+          name: floorist-operator-stage-test
```

Comment:
After trying this locally: the container inside the pod does not inherit the `namePrefix`. 
This is not that big of a problem in my opinion since the pod **does have** the prefix. 
If it's up to me I'd leave it as it is though.